engine/src/main/resources/org/hibernate/validator/ValidationMessages_pt_BR.properties

```diff
@@ -1,19 +1,25 @@
-javax.validation.constraints.DecimalMax.message  = deve ser menor ou igual a {value}
-javax.validation.constraints.DecimalMin.message  = deve ser maior ou igual a {value}
-javax.validation.constraints.Digits.message      = valor num\u00E9rico fora do limite (<{integer} d\u00EDgitos>.<{fraction} d\u00EDgitos> esperado)
-javax.validation.constraints.Email.message       = N\u00E3o \u00E9 um endere\u00E7o de e-mail
-javax.validation.constraints.Future.message      = deve estar no futuro
-javax.validation.constraints.Max.message         = deve ser menor ou igual a {value}
-javax.validation.constraints.Min.message         = deve ser maior ou igual a {value}
-javax.validation.constraints.NotBlank.message    = N\u00E3o pode estar em branco
-javax.validation.constraints.NotEmpty.message    = N\u00E3o pode estar vazio
-javax.validation.constraints.NotNull.message     = n\u00E3o pode ser nulo
-javax.validation.constraints.Null.message        = deve ser nulo
-javax.validation.constraints.Past.message        = deve estar no passado
-javax.validation.constraints.Pattern.message     = deve corresponder \u00E0 "{regexp}"
-javax.validation.constraints.Size.message        = tamanho deve estar entre {min} e {max}
+javax.validation.constraints.AssertFalse.message      = deve ser falso
+javax.validation.constraints.AssertTrue.message       = deve ser verdadeiro
+javax.validation.constraints.DecimalMax.message       = deve ser menor ou igual a {value}
+javax.validation.constraints.DecimalMin.message       = deve ser maior ou igual a {value}
+javax.validation.constraints.Digits.message           = valor num\u00E9rico fora do limite (<{integer} d\u00EDgitos>.<{fraction} d\u00EDgitos> esperado)
+javax.validation.constraints.Email.message            = N\u00E3o \u00E9 um endere\u00E7o de e-mail
+javax.validation.constraints.Future.message           = deve estar no futuro
+javax.validation.constraints.FutureOrPresent.message  = deve de ser uma data em presente ou em futuro
+javax.validation.constraints.Max.message              = deve ser menor ou igual a {value}
+javax.validation.constraints.Min.message              = deve ser maior ou igual a {value}
+javax.validation.constraints.Negative.message         = deve ser menor a 0
+javax.validation.constraints.NegativeOrZero.message   = deve ser menor ou igual a 0
+javax.validation.constraints.NotBlank.message         = N\u00E3o pode estar em branco
+javax.validation.constraints.NotEmpty.message         = N\u00E3o pode estar vazio
+javax.validation.constraints.NotNull.message          = n\u00E3o pode ser nulo
+javax.validation.constraints.Null.message             = deve ser nulo
+javax.validation.constraints.Past.message             = deve estar no passado
+javax.validation.constraints.PastOrPresent.message    = deve de ser uma data em passado ou em presente 
+javax.validation.constraints.Pattern.message          = deve corresponder \u00E0 "{regexp}"
+javax.validation.constraints.Positive.message         = deve ser maior a 0
```

Comment:
Hi, thanks for chiming in. Google Translate tells me

> Must be greater than 0

for both variants. Any specific reason for choosing one over the other? Thanks!